draft/Makefile

```diff
@@ -61,12 +63,8 @@ $(next).xml: $(draft).xml ../src/yang/iana-bfd-types.yang ../src/yang/ietf-bfd-i
 	sed -e"s/YYYY-MM-DD/$(shell date +%Y-%m-%d)/" ../src/yang/example-bfd-echo.yang > ../src/yang/example-bfd-echo\@$(shell date +%Y-%m-%d).yang
 	sed -e"s/YYYY-MM-DD/$(shell date +%Y-%m-%d)/" ../src/yang/example-bfd-client.yang > ../src/yang/example-bfd-client\@$(shell date +%Y-%m-%d).yang
 
-	cd ../src; ./gen-trees.sh; cd ..;
+	cd ../src; ./validate-and-gen-trees.sh; cd ..;
 	../src/insert-figures.sh $@ > tmp; mv tmp $@
-	rm ../src/yang/*-tree.txt
```

Comment:
Why are those files not being removed anymore?

Reply:
They are removed by the validate-and-gen-tree.sh script in the src directory.

Reply:
Got it